source/sass/components/nav.scss

```diff
@@ -1,254 +1,29 @@
-#primary-nav-container {
```

Comment:
#### What `nav.scss`  was before

It only contained rule set for `#primary-nav-container`.

#### What I did in this PR

- renamed the old `nav.scss` to `primary-nav.scss`
- created a new `nav.scss` and moved `.multipage-link` rules here (these rules are shared and used by various nav sections)